Xamarin.Forms.Core/ImageButton.cs

```diff
@@ -148,15 +152,15 @@ void IBorderElement.OnBorderColorPropertyChanged(Color oldValue, Color newValue)
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public void SendClicked() =>
```

Comment:
aren't those part of an interface, why don't you implement them explicitly ?

Reply:
These are just copied straight from *Button* and they existed before my initial PR.  In Button they are public so that's why they ended up public here.  It seems like it's an API that people use with Button type controls so they can "Send clicks" from code on the elements